service.watchedlist/addon.xml

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.watchedlist"
+	name="WatchedList"
+	version="1.2.1"
+	provider-name="schapplm">
+	<requires>
+		<import addon="xbmc.python" version="2.25.0"/>
+		<import addon="script.module.simplejson" version="2.0.10" />
+		<import addon="script.module.buggalo" version="1.1.3" /> <!-- min. version 1.1.3 for email report -->
+		<import addon="script.module.myconnpy" version="1.1.7" /> <!-- mySQL Connector for python. Default in XBMC since Frodo -->
+		<!--<import addon="script.module.pydevd" version="3.4.1"/> --> <!-- Required for remote debugging in Linux -->
+		<import addon="script.module.dropbox" version="8.4.0"/> <!-- For Dropbox synchronisation -->
+	</requires>
+	<extension point="xbmc.python.script" library="manual.py"> <!--Python code for manual start in the addons menu-->
+	  <provides>executable</provides> <!-- Addon is shown under Programs -->
+	</extension>
+	<extension point="xbmc.service" library="default.py" start="login" /> <!-- Python code for autostart -->
+  <extension point="xbmc.python.module" library="lib" /> <!-- entry to access the WatchedList from other addons -->
+	<extension point="xbmc.addon.metadata">
+		<summary lang="en">Save watched Movies and TV Episodes independent from filenames</summary>
```

Comment:
can you please change the language identifiers to en_gb and de_de? See: http://kodi.wiki/view/Addon.xml#.3Csummary.3E and http://kodi.wiki/view/language_support